docs/readthedocs/source/doc/Nano/Overview/pytorch_inference.md

```diff
@@ -22,6 +22,27 @@ For quantization, BigDL-Nano provides only post-training quantization in `Infere
 
```

Comment:
In line 14, we may just remove "Model conversion to 16-bit like BF16 is supported now."

Reply:
Have removed this sentence.